app/plugin_install.go

```diff
@@ -254,6 +257,25 @@ func (a *App) removePlugin(id string) *model.AppError {
 	return nil
 }
 
+func (a *App) removeSignatures(pluginId string) *model.AppError {
+	counter := 1
+	for {
```

Comment:
```
for counter := 1;; counter++ {
}
```
cleaner? :P

Reply:
Also I wonder if we should limit the number of plugin signature we can consume, otherwise it may lead to a DOD where a plugin that contains 99999999999 signatures can maybe cause havoc?

Reply:
If server contains that many signatures we are already doomed. Deleting them will not make things worse :)